rero_ils/modules/documents/jsonschemas/documents/document_uniform_title-v0.0.1.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
This whole field has been rewrited as field `work_access_point`. See L55 in CONVERSION.